pymatgen/io/vasp/inputs.py

```diff
@@ -244,9 +244,9 @@ def from_file(cls, filename, check_for_potcar=True, read_velocities=True, **kwar
 
         If all else fails, the code will just assign the first n elements in
         increasing atomic number, where n is the number of species, to the
-        Poscar. For example, H, He, Li, .... This will ensure at least a
-        unique element is assigned to each site and any analysis that does not
-        require specific elemental properties should work fine.
+        Poscar, where a warning would be issued. For example, H, He, Li, ....
+        This will ensure at least a unique element is assigned to each site and
+        any analysis that does not require specific elemental properties should work.
```

Comment:
I inserted a comment about warning here, so adjust the lines try not to make lines too long.